oxidauth-postgres/migrations/20240416170202_create_auth_keys.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE auth_keys (
+    id UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
+    user_id UUID FOREIGN KEY REFERENCES Users(id) NOT NULL,
+    key INTEGER[] NOT NULL,
```

Comment:
- column name might need to be more than just keys. 
- any downsides to integer[] type?